template/registry/number_test.go

```diff
@@ -87,4 +90,12 @@ func Test_Number(t *testing.T) {
 	assert = assertFormat(t, numberRegistryFunc,
 		map[string]any{"style": "percent", "minimumIntegerDigits": 3}, language.AmericanEnglish)
 	assert(0.01, "001%")
+
+	assert = assertFormat(t, numberRegistryFunc,
+		map[string]any{
+			"style":                    "percent",
+			"maximumFractionDigits":    5,
+			"maximumSignificantDigits": 4,
+		}, language.AmericanEnglish)
+	assert(0.12345, "12.34%")
```

Comment:
Please use some bigger number to show that the resulting number gets rounded not truncated
```suggestion
	assert(0.12346, "12.35%")
```

Reply:
I'd like to keep it as it is - to emphasize it does not always round up.

I have not figured out how rounding works in the `number` package.